src/anomalib/trainer/utils/thresholder.py

```diff
@@ -86,17 +70,45 @@ def update(self, outputs: STEP_OUTPUT) -> None:
         Args:
             outputs (STEP_OUTPUT): Step outputs.
         """
-        if self.threshold_method == ThresholdMethod.ADAPTIVE:
-            self._update_thresholds(self.trainer.image_threshold, self.trainer.pixel_threshold, outputs)
-
-    @staticmethod
-    def _update_thresholds(
-        image_metric: AnomalyScoreThreshold,
-        pixel_metric: AnomalyScoreThreshold,
-        outputs: STEP_OUTPUT,
-    ) -> None:
+        image_metric = self.trainer.image_threshold
+        pixel_metric = self.trainer.pixel_threshold
```

Comment:
can you remind me why we assign threshold to metric variable?

Reply:
Yeah I think image_threshold and pixel_threshold might be better names. I guess initially the `_update_thresholds` method also computed the metric values.